packages/website/docs/_components_pages/fiori/Search/Search.mdx

```diff
@@ -19,4 +20,9 @@ A Search component with enabled scope and items and filtering by scope.
 ### Filtering
 The examples shows Search Message Area, Search Action, filtering per user input, typeahead and highlighting suggestions per user input.
 
-<Advanced />
\ No newline at end of file
+<Advanced />
+
+### Byline items
+The examples shows how to display byline items with an image and description
```

Comment:
Is it one example or several? It should either be: "The examples show" or "The example shows" how to display byline items with an image and description.